### ISSUE TITLE
Can't bind to 'brmasker' since it isn't a known property of 'ion-input'.

### ISSUE THREAD

**Issue by garauz23** [amarkes/br-mask#14]
I added on app.module.ts 
import { BrMaskerModule } from "br-mask";

@NgModule({
  declarations: [AppComponent],
  entryComponents: [],
  imports: [
    ...
    , BrMaskerModule
  ],...

but i still getting the following message: 
Can't bind to 'brmasker' since it isn't a known property of 'ion-input'.

**Comment by vinicius-santos** [comment]
> I added on app.module.ts
> import { BrMaskerModule } from "br-mask";
> 
> @NgModule({
> declarations: [AppComponent],
> entryComponents: [],
> imports: [
> ...
> , BrMaskerModule
> ],...
> 
> but i still getting the following message:
> Can't bind to 'brmasker' since it isn't a known property of 'ion-input'.

Use this, but not in the module module class, but in the module where you will use the mask

  imports: [
    BrMaskerModule
  ],
  
  providers: [
    StatusBar,
    BrMaskDirective
 
  ],

**Comment by garauz23** [comment]
> > I added on app.module.ts
> > import { BrMaskerModule } from "br-mask";
> > @NgModule({
> > declarations: [AppComponent],
> > entryComponents: [],
> > imports: [
> > ...
> > , BrMaskerModule
> > ],...
> > but i still getting the following message:
> > Can't bind to 'brmasker' since it isn't a known property of 'ion-input'.
> 
> Use this, but not in the module module class, but in the module where you will use the mask
> 
> imports: [
> BrMaskerModule
> ],
> 
> providers: [
> StatusBar,
> BrMaskDirective
> 
> ],

I undestand, I dont need to added to app.module.ts, I just need to add BrMaskerModule as import and BrMaskDirective as provider on mypage.module.ts, 

Am I correct?

**Comment by kapilraghuwanshi** [comment]
Yes, just add directive and module in your own sub module not in app.module.ts. It is working.
